lightkube/core/client.py

```diff
@@ -82,24 +82,26 @@ def delete(self, res, name: str, *, namespace: str = None, grace_period: int = N
             * 'CascadeType.ORPHAN' - orphan the dependents;
             * 'CascadeType.BACKGROUND' - allow the garbage collector to delete the dependents in the background;
             * 'CascadeType.FOREGROUND' - a cascading policy that deletes all dependents in the foreground.
+        * **dry_run** - *(optional)* Submit request to server without persisting the resource.
```

Comment:
The current description seems to apply for `create` but doesn't make sense for `delete` or the other verbs.

```suggestion
        * **dry_run** - *(optional)* Submit request to server without persisting the change.
```

Alternatively this is the way it's documented in the API (which I think is even more accurate):

```suggestion
        * **dry_run** - *(optional)* When present, indicates that modifications should not be persisted..
```